x/chainlink/client/cli/txFeed.go

```diff
@@ -1,14 +1,88 @@
 package cli
 
 import (
+	"errors"
+	"strconv"
+	"strings"
+
 	"github.com/ChainSafe/chainlink-cosmos/x/chainlink/types"
 	"github.com/cosmos/cosmos-sdk/client"
 	"github.com/cosmos/cosmos-sdk/client/flags"
 	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 	"github.com/spf13/cobra"
-	"strings"
 )
 
+func CmdAddFeed() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "addFeed [feedId] [feedOwnerAddress] [submissionCount] [heartbeatTrigger] [deviationThresholdTrigger] [initDataProviderList]",
+		Short: "Add new feed. Signer must be the existing module owner.",
+		Args:  cobra.MinimumNArgs(6),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			argsFeedId := strings.TrimSpace(args[0])
+			argsFeedOwnerAddr := strings.TrimSpace(args[1])
+			argsSubmissionCount := strings.TrimSpace(args[2])
+			argsHeartbeatTrigger := strings.TrimSpace(args[3])
+			argsDeviationThresholdTrigger := strings.TrimSpace(args[4])
```

Comment:
i dont think we need to trime space for these

Reply:
correct